### PR TITLE
#4 | Added Jumbotron

### PR DESCRIPTION
**Pull request by ThijmenMaus**
See Discord `#main-website` for content changes

### CODE REVIEW

File: src/components/Navbar/Navbar.module.scss [hunk 1]
```diff
@@ -1,19 +1,24 @@
 @import '../../styles/variables', '../../styles/util';
 
-/// The root styles of the navigation bar
-.root {
+/// Styles of the navigation bar
```
Comment (ThijmenMaus) [inline src/components/Navbar/Navbar.module.scss]: Styles of -> Styles for

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
 
+import { Jumbotron } from '~components';
 import { NAVIGATION_LINKS } from '~constants';
 import { parseElementId } from '~util';
```
Comment (ThijmenMaus) [inline src/pages/index.tsx:6]: JSDoc on index page

File: src/styles/_util.scss [hunk 3]
```diff
@@ -8,7 +8,44 @@
 }
 
 /// Fix row double negative margin
```
Comment (ThijmenMaus) [inline src/styles/_util.scss]: Consistent sassdoc (A function that ....)

**Review by Paludeto — APPROVED** [review]
Responsive and pretty overall, not too sure about the code because I'm a dummy, but it looks clean to me.

**Pull request merged**